tests/controllers/hibp.test.js

```diff
@@ -47,7 +73,7 @@ test("notify POST with unknown breach should throw error", async () => {
   const testPrefix = testHash.slice(0, 6).toUpperCase();
   const testSuffix = testHash.slice(6).toUpperCase();
 
-  const mockRequest = { body: { hashPrefix: testPrefix, hashSuffixes: [testSuffix], breachName: "Test" }, app: { locals: { breaches: [] } } };
+  const mockRequest = { token: "unsafe-default-token-for-dev", body: { hashPrefix: testPrefix, hashSuffixes: [testSuffix], breachName: "Test" }, app: { locals: { breaches: [] } } };
```

Comment:
not sure if these two rascals should be `AppConstants.HIBP_NOTIFY_TOKEN`, but probably doesn't matter since they're hardcoded for dev/testing.